common-lib/src/main/scala/com/gu/mediaservice/lib/config/CommonConfig.scala

```diff
@@ -29,30 +20,36 @@ trait CommonConfig extends AwsClientBuilderUtils {
 
   final val sessionId = UUID.randomUUID().toString
 
-  override val awsRegion: String = properties.getOrElse("aws.region", "eu-west-1")
+  // TODO:SAH - remove these and favour explicit config for anything that is derived from here
+  val stage: String = string(GridConfigLoader.STAGE_KEY)
```

Comment:
Does this mean the HOCON config files can have a `grid.stage` property, which overrides any value read from `/etc/grid/stage`?

Reply:
Yes. I'm thinking about deprecating `/etc/grid/stage` and keeping config all in one place. Can't think of a good reason not too.